packages/cli/test/integration/public-api/endpoints-with-scopes-enabled.test.ts

```diff
@@ -0,0 +1,1694 @@
+import { Container } from '@n8n/di';
```

Comment:
This basically enables the api keys scopes feature and run all the tests that we already have (ported them here). Perhaps it's better to test all the scenarios where it should fail? so for each route, hit it with all scopes, but the one needed and make sure it fails. I guess here only are testing the middleware and nothing else, as everything else is the same.

Reply:
The middleware is tested and the scopes are assigned declarative, so maybe we don't really need all of these tests. But then again they don't do any harm for now.